bcrhp/src/bcrhp/pages/steps/Step4_SiteNames.vue

```diff
@@ -39,55 +46,34 @@ const isValid = () => {
     let valid = true;
 
     for (const field of Object.values(fields) as Array<Ref>) {
-        valid = validateField(field?.value.$el as HTMLInputElement) && valid;
+        valid =
+            validateField(field?.value.$el as FormFieldResolverOptions) &&
+            valid;
     }
     return valid;
 };
 
-const updateAddOtherState = function () {
-    addOtherNameDisabled.value =
-        otherName.value.length < 1 ||
-        heritageSiteRef.value.otherNames.length > 4;
+const resolver = function (e: FormFieldResolverOptions): Record<string, any> {
+    return validateField(e as FormFieldResolverOptions);
 };
 
-const valueUpdated = function (value: string | undefined) {
-    console.log(`valueUpdated: ${value}`);
-    updateAddOtherState();
```

Comment:
This is no longer being called when entering data in to the Other Name field, so the Add button doesn't become active when the field has data in it. Maybe we need to put a watch statement in on the Other Name data to update the state.